ircbot/plugins/transfer.py

```diff
@@ -0,0 +1,52 @@
+import re
+
+from ircbot import bot
+from redis import StrictRedis
+
+@bot.command('transfer')
+def transfer(bot, channel, sender, args):
+    """"command to transfer karma & butts from one term to another"""
+    if sender.lower() == bot.config['System']['owner']:
+        redis = StrictRedis.from_url(bot.config['System']['redis_url'])
+
+        if len(args) == 2:
+            old_name = args[0].lower()
+            new_name = args[1].lower()
+        else:
+            string = ""
+            strings = []
+            for word in args:
+                string = "{} {}".format(string, word)
```

Comment:
What are you doing here, exactly? Are you just joining all the `args` together?

Reply:
yes, so the regex can deal with it

Reply:
Python has a `join` function for lists to join them together into strings. I'd recommand changing this to `strings = args.join(" ")`.

Reply:
oh yeah it does doesn't it, ok I'll add that